test/prototypes/GatewayEVMIntegration.spec.ts

```diff
@@ -0,0 +1,320 @@
+import { expect } from "chai";
+import { BigNumber, Contract } from "ethers";
+import { ethers, upgrades } from "hardhat";
+
+describe("GatewayEVM and Receiver", function () {
```

Comment:
```suggestion
describe("GatewayEVM inbound", function () {
```
For consistency with other test section